pkg/api/v1/conversion.go

```diff
@@ -346,13 +351,20 @@ func Convert_v1_PodTemplateSpec_To_api_PodTemplateSpec(in *PodTemplateSpec, out
 	if err := autoConvert_v1_PodTemplateSpec_To_api_PodTemplateSpec(in, out, s); err != nil {
 		return err
 	}
+
+	// TODO: when seccomp is out of alpha this can be removed
+	internalConvertV1SeccompToApiSeccomp(in.Annotations, &out.Spec)
```

Comment:
Good spot that we need this here in PodTemplateSpec as well. Are there more those cases or do all DaemonSet, ReplicaSet, PetSet use the same PodTemplateSpec (I hope so)?